src/BuildScriptGenerator/Node/NodePlatform.cs

```diff
@@ -483,10 +488,8 @@ private string GetNpmVersion(dynamic packageJson)
             if (!string.IsNullOrWhiteSpace(npmVersionRange))
             {
                 var supportedNpmVersions = _nodeVersionProvider.SupportedNpmVersions;
-                npmVersion = SemanticVersionResolver.GetMaxSatisfyingVersion(
-                    npmVersionRange,
-                    supportedNpmVersions);
-                if (string.IsNullOrWhiteSpace(npmVersion))
+                npmVersion = SemanticVersionResolver.GetMaxSatisfyingVersion(npmVersionRange, supportedNpmVersions);
+                if (string.IsNullOrEmpty(npmVersion))
```

Comment:
why this change?

Reply:
because it fits in less than 120 characters